usafacts/tests/test_geo.py

```diff
@@ -108,19 +103,19 @@ def test_hrr(self):
                 "population": [100, 2100, 300, 25],
             }
         )
-
-        new_df = geo_map(df, "hrr", MAP_DF, SENSOR)
-
-        exp_incidence = np.array([13, 27]) / np.array([25, 2500]) * 100000
-        exp_cprop = np.array([60, 165]) / np.array([25, 2500]) * 100000
-
-        assert (new_df["geo_id"].values == [110, 147]).all()
-        assert (new_df["timestamp"].values == ["2020-02-15", "2020-02-15"]).all()
-        assert new_df["new_counts"].values == pytest.approx([13.0, 27.0])
-        assert new_df["cumulative_counts"].values == pytest.approx([60, 165])
-        assert new_df["population"].values == pytest.approx([25, 2500])
-        assert new_df["incidence"].values == pytest.approx(exp_incidence)
-        assert new_df["cumulative_prop"].values == pytest.approx(exp_cprop)
+        new_df = geo_map(df, "hrr", SENSOR)
+        pd._testing.assert_frame_equal(
```

Comment:
Why is this `pd._testing` rather than just `pd.testing`?

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html

Reply:
oops I copy pasted the first thing i found on google and guess I found something else. Thanks for catching, should be fixed now.